spec/mailers/message_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe MessageMailer, type: :mailer do
+  describe 'create' do
+    let(:message) { Message.new('Lucas', 'lucas@email.com', 'test email message') }
+    let(:mail) { MessageMailer.new_message(message.name, message.email, message.message) }
+
+    it 'renders the subject' do
+      expect(mail.subject).to eql("Message from #{message.name} on bastienbarbé !")
+    end
+
+    it 'renders the receiver email' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.